### PR TITLE
Adding demo for namespace recommendations

### PR DESCRIPTION
**Pull request by shekhar316**
# Namespace Recommendations Demo
Image: http://quay.io/rh-ee-shesaxen/autotune/kruize-namespacetest-v5

Note: 

- Run local_monitoring_namespace_demo script to create experiment and generate recommendations for namespace.

### CODE REVIEW

**Comment by shekhar316** [comment]
@chandrams created a different script for namespace demo and to use test-multiple-import namespace.

File: monitoring/local_monitoring/local_monitoring_demo_namespace.sh [hunk 1]
```diff
@@ -0,0 +1,403 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, 2022 Red Hat, IBM Corporation and others.
```
Comment (chandrams) [inline monitoring/local_monitoring/local_monitoring_demo_namespace.sh:3]: @shekhar316 - Can you add a cleanup or terminate routine to undo the setup say with -t option and update the readme.

Comment (chandrams) [reply monitoring/local_monitoring/local_monitoring_demo_namespace.sh:3]: Update the copyright year

**Comment by chandrams** [comment]
In the recommendations json output, can you remove the namespace name as it repeated and use underscore in namespaceRecommendations to keep it consistent

```
  "kubernetes_objects": [
      {
        "namespace": "test-multiple-import",
        "namespaceAPIObject": {
          "namespace_name": "test-multiple-import",
          "namespaceRecommendations": {
```

**Comment by shekhar316** [comment]
> In the recommendations json output, can you remove the namespace name as it repeated and use underscore in namespaceRecommendations to keep it consistent
> 
> ```
>   "kubernetes_objects": [
>       {
>         "namespace": "test-multiple-import",
>         "namespaceAPIObject": {
>           "namespace_name": "test-multiple-import",
>           "namespaceRecommendations": {
> ```

Update the format according to the document.

File: monitoring/local_monitoring/local_monitoring_demo_namespace.sh (hunk 1 above)
Comment (shekhar316) [reply monitoring/local_monitoring/local_monitoring_demo_namespace.sh]: Updated the copyright year. 
Now, deleting the all repos and namespace quota applied with -t option.

**Comment by chandrams** [comment]
@shekhar316 - Can you resolve the conflicts & update documentation

**Comment by dinogun** [comment]
@shekhar316 Please fix the conflicts

**Comment by shekhar316** [comment]
> @shekhar316 Please fix the conflicts

Resolved the conflicts.

**Comment by dinogun** [comment]
@shekhar316 I would suggest to not have a separate script for namespace, it is better if this is part of the `local_monitoring` one itself

**Comment by shekhar316** [comment]
> @shekhar316 I would suggest to not have a separate script for namespace, it is better if this is part of the `local_monitoring` one itself

Sure, I have updated the demo to use the single script.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**